### DISCUSSION TITLE
Bookmark All Tabs option in context menu?

### DISCUSSION THREAD

**Discussion by paulri** [piroor/treestyletab#3084] [General]
When I use TST, I often like to bookmark all tabs open in a given window of FF, before closing the window. Is there a way to do this? I poked around in the options for the context menu but didn't see any way to do this.

**Comment by irvinm** [comment]
You can do this natively in Firefox:

> To bookmark all open tabs at once: Right-click on any of the tabs, choose Select All Tabs from the context menu, then right-click on any tab and select Bookmark Tabs…. Give the new bookmarks folder a name and choose which folder to store it in. Click Add Bookmarks to finish.

You can also do it with TST:

> Right-click on a tab, select "Select All Tabs", then right-click on a tab again and choice "Bookmark tabs..."

**Reply by paulri** [reply]
OK, I see that. Two operations, selecting and then bookmarking. 

Thanks.
